modules/installing-ocp-agent-inputs.adoc

```diff
@@ -159,4 +169,25 @@ You must provide the rendezvous IP address when you do not specify at least one
 <2> Optional: Host configuration. The number of hosts defined must not exceed the total number of hosts defined in the `install-config.yaml` file, which is the sum of the values of the `compute.replicas` and `controlPlane.replicas` parameters.
 <3> Optional: Overrides the hostname obtained from either the Dynamic Host Configuration Protocol (DHCP) or a reverse DNS lookup. Each host must have a unique hostname supplied by one of these methods.
 <4> Enables provisioning of the Red Hat Enterprise Linux CoreOS (RHCOS) image to a particular device. It examines the devices in the order it discovers them, and compares the discovered values with the hint values. It uses the first discovered device that matches the hint value.
-<5> Optional: Configures the network interface of a host in NMState format.
\ No newline at end of file
+<5> Optional: Configures the network interface of a host in NMState format.
+
+ifdef::pxe-boot[]
+
+. Optional: To create an iPXE script, add the `bootArtifactsBaseURL` to the `agent-config.yaml` file:
++
+[source,yaml]
+----
+apiVersion: v1alpha1
```

Comment:
```suggestion
apiVersion: v1beta1
```
This [PR](https://github.com/openshift/installer/pull/6840/files) includes the changes.